src/unix/atomic-ops.h

```diff
@@ -33,6 +33,10 @@ UV_UNUSED(static int cmpxchgi(int* ptr, int oldval, int newval)) {
                         : "r" (newval), "0" (oldval)
                         : "memory");
   return out;
+#elif defined(_AIX) && (defined(__xlC__) || defined(__IBMC__) || defined(__IBMCPP__))
```

Comment:
http://beefchunk.com/documentation/lang/c/pre-defined-c/precomp.html
Looks like it is. I just used all of them. Though happy to make it just __xlC__.